helm/loki/values.yaml

```diff
@@ -179,10 +179,7 @@ loki:
       #customWriteUrl: http://loki-multi-tenant-proxy.default.svc.cluster.local:3101
       #customBackendUrl: http://loki-multi-tenant-proxy.default.svc.cluster.local:3102
     autoscaling:
-      enabled: true
-      minReplicas: 2
-      targetCPUUtilizationPercentage: 90
-      targetMemoryUtilizationPercentage: 90
+      enabled: false
```

Comment:
How does disabling autoscaling here enable keda?
Also, maybe we want to keep sane default autoscaling values in case we or anyone using our chart want to enable it again?

Reply:
You're right my bad, I forgot to enable back even though that's the title of the PR :facepalm: 

> Also, maybe we want to keep sane default autoscaling values in case we or anyone using our chart want to enable it again?

That's a good point, I'm gonna put those back.